sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslationError.cs

```diff
@@ -35,6 +46,8 @@ internal DocumentTranslationError(DocumentTranslationErrorCode errorCode, string
                 Message = message;
                 Target = target;
             }
+
+            InnerError = null;
```

Comment:
Do we need the `InnerError` property if we're just making it null?

Reply:
I'm assuming it's generated by CodeGen. It can't be helped then.

Reply:
wasn't able to remove it, so need to leave it. and because it is a struct I need to assign a value to it, so always assigning null.